website/community/product-steering/_index.md

```diff
@@ -0,0 +1,55 @@
+Product Steering provides strategic direction for major product increments. Product ideas can be submitted to the Product Steering Committee as proposals. The proposer must provide a brief document outlining the idea. Ideally, if applicable, a prototype should be included to demonstrate feasibility, but no full implementation or design document is expected (or desired) at this stage.
+
+The committee will schedule a Product Steering meeting, where the proposer presents the proposal. Committee members, interested contributors, and affected sponsors will challenge and discuss the idea. This meeting will be recorded.
+
+Following this, a private go/no-go decision meeting with the affected sponsors will take place. This second meeting will not be recorded, but the outcome will be documented and shared.
+
+Only proposals requiring significant development effort or broad cross-team impact will be considered.
+
+### Why Do We Need It?
+We aim to align sponsored development capacity with Gardener's strategic vision and priorities. Investments should deliver value either to Gardener's users and operators or to the project and its overall reach.
+
+### Who is on The Product Steering Committee?
+The committee is elected by the project's active contributing sponsors and consists of a small, odd number of members (at least three) to ensure majority decision-making. Currently, the committee members are:
+- [Johannes Scheerer](https://github.com/scheererj)
+- [Rafael Franzke](https://github.com/rfranzke)
+- [Vedran Lerenc](https://github.com/vlerenc)
+
+### How Does It Work?
+#### 1. Proposal Submission
+- A proposer drafts a **2-5 page product proposal** detailing the idea and shares it with any member of the Product Steering Committee.
+- The proposal is linked on this community page, and a **meeting date** is scheduled **1-4 weeks out** (at least one week for review and preparation but no more than four weeks to ensure timely decisions).
+
+#### 2. Participation
+- Those who wish to **actively challenge or discuss** the proposal can **request an invitation** from any committee member.
+- Others are encouraged to **watch the recording** instead of attending live.
+
+#### 3. Meeting Process
+- The proposer presents the product proposal.
+- Attendees provide feedback, ask questions, and challenge the document.
+- The session is recorded for transparency.
+
+#### 4. Decision Process
+- A **private meeting** with sponsors follows to make a **go/no-go decision**.
+- The final decision is **documented on this community page**.
+
+### Key Considerations for Topics
+A topic qualifies for Product Steering if it:
+- Requires **significant development effort** (≥6 person-months), OR
+- Has a **moderate to high impact** on other teams (e.g., landscape stability, operations).
+
+<hr/>
+
+## Planned Topics
+
+- **Recommender (CA Replacement)** (no paper/date yet)
+- **Cluster API** (no paper/date yet)
+- **Control Plane Live Migration** (no paper/date yet)
```

Comment:
You could link https://github.com/gardener/gardener/issues/10686

Reply:
Thanks